docs/docs/administer/logs.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Logs
+sidebar_position: 8
+description: Firezone's WireGuard®-based remote access platform stores debug logs and configuration, and audit logs.
+---
+
+:::note
+This guide is written for Docker based deployments of Firezone
+:::
+
+## What Firezone Logs
+
+Docker deployments of Firezone consist of 3 running containers:
+
+| Container         | Function      | Example logs                                  |
+|-------------------|---------------|-----------------------------------------------|
+| ubuntu-firezone-1 | Web portal    | HTTP requests received and responses provided |
+| ubuntu-postgres-1 | Database      |                                               |
+| ubuntu-caddy-1    | Reverse proxy |                                               |
+
+Each container stores logs as a JSON file on the host machine. These files are found at
+`var/lib/docker/containers/{CONTAINER_ID}/{CONTAINER_ID}-json.log` and can be shown through the
+`docker logs {CONTAINER}` command.
+
+## Managing and Configuring Logs
+
+By default, Firezone uses the `json-file` logging driver without
+[additional configuration](https://docs.docker.com/config/containers/logging/json-file/). This means
+logs from each container is stored separately in a file format designed to be exclusively accessed by
+the Docker daemon. Log rotation is not enabled, so logs on the host can build up and consume excess
+storage space.
+
+For production deployments of Firezone you may want to configure how logs are collected and stored. Docker
+provides [multiple mechanisms](https://docs.docker.com/config/containers/logging/configure/)
+to collect information from running containers and services.
+
+Examples of popular plugins, configurations, and use cases are:
+
+* Export container logs to
+[Splunk](https://docs.docker.com/config/containers/logging/splunk/)
+or
+[Google Cloud Logging](https://docs.docker.com/config/containers/logging/gcplogs/)
+* Enable log rotation and max file size
+* [Customize log driver output](https://docs.docker.com/config/containers/logging/log_tags/)
+
+## Audit Logs
+
+:::note
+Audit logs are in early Beta on the Enterprise plan.
+[Contact us](https://www.firezone.dev/contact/sales)
+to learn more.
+:::
+
+Firezone maintains two types of logs tied to user identity: configuration logs and network activity logs.
+
+Configuration logs track events related to the configuration of Firezone itself, either by admins or users.
+Access logs track connections to protected resources made by users of your network.
+
+Logged events are recorded as JSON objects and stored in the database. These are accessible via a
```

Comment:
Just want to clarify that the Docker described above are not stored in the database. So we have 3 kinds of logs it seems:

* Infrastructure / deployment logs
* Configuration change logs
* Network activity logs

Thoughts on how to segregate or otherwise draw boundaries between these? I foresee confusion on the horizon.

Reply:
Having three separate sections on 1 page could work. I changed "What Firezone logs" to "Deployment logs" in the interim before we add config + network activity logs.

We could use [admonitions](https://docusaurus.io/docs/markdown-features/admonitions) for callouts (i.e. enterprise plan, database vs. json storage)

Reply:
They're going to diverge quite substantially in the following ways:

* What they're used for
* The integrity level needed
* Where and how they're stored
* Data retention policies / periods

Longer retention periods in managed SaaS products typically correspond to higher price tiers (this would be a silly artificial limitation misaligned with our pricing ethos, but the example stands)

Reply:
Ok, agreed. Our current logs are more useful for debugging your deployment. Seems more in line with 'troubleshooting' and written for admins.

Let's split out audit logs and put it in 'references' or 'user guides' when we release that feature. It's more of a feature than a deployment detail.  

What do you think?

Reply:
👍